ospd/ospd.py

```diff
@@ -651,8 +651,16 @@ def sort_host_finished(
             progress = current_hosts.get(finished_host)
             if progress == ScanProgress.FINISHED:
                 alive_hosts.append(finished_host)
-            if progress == ScanProgress.DEAD_HOST:
+            elif progress == ScanProgress.DEAD_HOST:
                 dead_hosts.append(finished_host)
+            else:
+                logger.warning(
+                    'The host %s is considered dead or finished, but '
+                    'its progress is still %d. This can lead to '
+                    'interrupted scan.',
```

Comment:
What's the consequence of this? Does it have some impact? What's the user supposed to do?

Reply:
The scan just ends with an interrupted scan. At this point in OSPD, we don't know what caused the interrupted scan. Before, the possible cause of an interrupted openvas scan  were  a duplicated host in target, unresolved host name, a race condition, and others. Many of the issues were already addressed, but as it depends on the scanner wrapper as well, IMHO this message must be generic. Probably I should set the log level to debug, since the user can't do so much in this cases.

Reply:
I changed the message log level with https://github.com/greenbone/ospd/pull/363/commits/7fac988c830ce97c7e6ba01d68193b708ffb0966